Dijkstra Algorithm/README.md

```diff
@@ -0,0 +1,45 @@
+# Dijkstra-algorithm
+
+## About this repository
+This repository contains to playgrounds:
+* To understand how does this algorithm works, I have created **VisualizedDijkstra.playground.** It works in auto and interactive modes. Moreover there are play/pause/stop buttons.
+* If you need only realization of the algorithm without visualization then run **Dijkstra.playground.** It contains necessary classes and couple functions to create random graph for algorithm testing.
+
+## Demo video
+Click the link: [YouTube](https://youtu.be/PPESI7et0cQ)
+
+## Screenshots
+<img src="Images/screenshot1.jpg" height="400" />
+
+## Dijkstra's algorithm explanation
+
+<img src="Images/Dijkstra_Animation.gif" height="250" />
+
+[Wikipedia](https://en.wikipedia.org/wiki/Dijkstra%27s_algorithm)'s explanation:
```

Comment:
Please don't just copy from Wikipedia. If it helps, you can quote a small paragraph from it, but most of the writing should be in your own words. 

What you've put here is the procedure of Dijkstra's algorithm. Before talking about the procedure, you need to talk about what the algorithm is, and what it will solve. I suggest you take some inspiration from this article: https://github.com/raywenderlich/swift-algorithm-club/tree/master/Merge%20Sort